db/organizations/loggly.eno

```diff
@@ -1,11 +1,12 @@
-name: Loggly
-website_url: https://www.loggly.com/
-privacy_policy_url: https://www.loggly.com/about/privacy-policy/
-privacy_contact: info@loggly.com
-country: 
-description: "Loggly is a San Francisco startup developing the world's most popular cloud based logging platform. We have built a highly scalable log management service as a platform which provides great value for the entire organization."
+name: SolarWinds
```

Comment:
Just noticed that both loggly and pingdom got acquired by SolarWinds.

To avoid blocking the rest of the PR, I split it out here and will look into it later: https://github.com/ghostery/trackerdb/issues/176

For now, please keep the original Loggly values, except for the notes.